app/models/reports/trainee_report.rb

```diff
@@ -22,8 +22,6 @@ def initialize(trainee)
              :first_names,
              :middle_names,
              :trn,
-             :withdraw_reasons_details,
-             :withdraw_reasons_dfe_details,
```

Comment:
What do we show for older trainees that used those fields?

Reply:
Well, we're not showing these on the front end any more, does it make sense to show them in the report? I'll drop a question on the ticket to get a steer to see if we should include them still.

Reply:
Yeah I'm not sure as well hence asking as otherwise we end up going in circles... ie next week we need to backtrack and add it back in